samples/client/petstore/csharp-netcore/OpenAPIClient-ConditionalSerialization/src/Org.OpenAPITools/Api/FakeApi.cs

```diff
@@ -2330,7 +2330,6 @@ public Org.OpenAPITools.Client.ApiResponse<Object> TestEndpointParametersWithHtt
             localVarRequestOptions.FormParameters.Add("byte", Org.OpenAPITools.Client.ClientUtils.ParameterToString(_byte)); // form parameter
             if (binary != null)
             {
-                localVarRequestOptions.FileParameters.Add("binary", binary);
```

Comment:
@iBicha looks like this line shouldn't be removed, right?

Reply:
The version that was implemented does not support file parameters, because they are not supported on some platforms (mostly NDA platforms like Nintendo/Sony/Microsoft) and will get rejected if not handled correctly.

So for the Unity version it is normal to see this line removed. But are these samples Unity specific?

Reply:
I think you're right, something is off, let me take a look

Reply:
"supportsFileParameters" was not synched, causing it to be off all the time. Fixed!